src/Constants.ts

```diff
@@ -229,10 +229,16 @@ export const ClassNames = {
     MONGODB: 'MONGODB',
     ELASTICSEARCH: 'ELASTICSEARCH',
     MYSQL: 'MYSQL',
+    EVENTBRIDGE: 'AWS-EventBridge',
 };
 
 export const AWS_SERVICE_REQUEST = 'AWSServiceRequest';
 
+export const AwsEventBridgeTags = {
+    SERVICE_REQUEST: 'AWSEventBridgeRequest',
+    EVENT_BUS_NAME: 'event.bus.name',
```

Comment:
I think, the `EVENT_BUS_NAME` tag name should be `aws.eventbridge.eventbus.name`